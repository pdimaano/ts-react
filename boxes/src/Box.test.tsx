import React from "react";
import { render } from "@testing-library/react";
import Box from "./Box";

const mock = () => { };

it("renders without crashing", function () {
    render(<Box id="1" width={10} height={10} backgroundColor="pink" remove={mock} />);
});