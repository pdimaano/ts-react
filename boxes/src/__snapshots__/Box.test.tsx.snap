// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Box"
  >
    <div
      class="Box-box"
      style="height: 10em; width: 10em; background-color: pink;"
    />
    <button
      class="Box-removeBtn"
    >
      Remove The Box!
    </button>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot with default properties 1`] = `
<DocumentFragment>
  <div
    class="Box"
  >
    <div
      class="Box-box"
      style="height: 10em; width: 10em; background-color: pink;"
    />
    <button
      class="Box-removeBtn"
    >
      Remove The Box!
    </button>
  </div>
</DocumentFragment>
`;
